src/components/Settings/People/People.component.js

```diff
@@ -95,126 +95,118 @@ const People = ({
         disableSubmit={!isLogged}
       >
         <Paper>
-          <List>
-            <ListItem>
-              <div className="Settings__UserIcon__Container">
-                <UserIcon />
-              </div>
-              <ListItemText primary={name} />
-              <ListItemSecondaryAction className="Settings--secondaryAction">
+          <Box px={2} py={2}>
+            <Box
+              display="flex"
+              flexDirection="column"
+              alignItems="center"
+              mb={4}
+            >
+              <UserIcon />
+              <Box display="flex" justifyContent="center" width="100%">
                 <Button
-                  disabled={!isLogged}
                   variant="outlined"
                   color="primary"
-                  onClick={logout}
+                  onClick={isLogged ? logout : login}
```

Comment:
Hey @KevinDussel Sorry for the delay. I think the login action is not necessary here. only redirect the user to `'/login-signup'` as you do it on the line :112 of this file